pkg/cable/wireguard/getconnections.go

```diff
@@ -141,9 +143,25 @@ func peerTrafficDelta(c *v1.Connection, key string, newVal int64) int64 {
 	return 0
 }
 
-// Save backendConfig[key]
-func savePeerTraffic(c *v1.Connection, lc, tx, rx int64) {
+// Save backendConfig[key] and export the metrics to prometheus
+func saveAndExportPeerTraffic(c *v1.Connection, lc, tx, rx int64) {
 	c.Endpoint.BackendConfig[lastChecked] = strconv.FormatInt(lc, 10)
 	c.Endpoint.BackendConfig[transmitBytes] = strconv.FormatInt(tx, 10)
 	c.Endpoint.BackendConfig[receiveBytes] = strconv.FormatInt(rx, 10)
+
+	endpointLabels := getLabelsFromEndpoint(&c.Endpoint)
+	timeCreated, _ := strconv.ParseInt(c.Endpoint.BackendConfig[timeCreated], 10, 64)
+	timeAlive := float64((time.Now().UnixNano() - timeCreated) / int64(time.Second))
+
+	cable.ConnectionTxBytes.With(endpointLabels).Set(float64(rx))
+	cable.ConnectionRxBytes.With(endpointLabels).Set(float64(tx))
+	cable.ConnectionActivationStatus.With(endpointLabels).Set(1)
+	cable.ConnectionUptimeDurationSeconds.With(endpointLabels).Set(timeAlive)
+}
+
+func getLabelsFromEndpoint(e *v1.EndpointSpec) prometheus.Labels {
+	return prometheus.Labels{"clusterID": e.ClusterID,
```

Comment:
Thanks for handling my comments, I propose that we stick to one label format:

cluster_id, private_ip (instead of camelCase format)

Reply:
the current shared layer use `under_scores` as the labels and metrics names conventions.